front/src/components/Chat/JoinLeave.tsx

```diff
@@ -128,16 +128,17 @@ export function LeaveChannel(props: { channel: IChannel }) {
 				setTimeout(() => {
 					if (swal && swal.close)
 					  swal.close()
-				  }, 1000);
+				  }, 600);
 			}
 		})
 		socket.on("leaveChannelFailed", (err_message) => {
 			if (currentUser !== undefined) {
 				swal(err_message, "error");
-				setTimeout(() => {
-					if (swal && swal.close)
-					  swal.close()
-				  }, 1000);
+				// setTimeout(() => {
+				// 	if (swal && swal.close)
+				// 	  swal.close()
+				//   }, 1000);
+				//sert a rien ?
```

Comment:
yeah, i agree we don't need this
I think I put like this in JoinLeave without thinking too much lol